### ISSUE TITLE
find_files always returns no result

### ISSUE THREAD

**Issue by nuevasindustriasdelafantasia** [nvim-telescope/telescope.nvim#736]
### Description
Running the command ``` :Telescope find_files``` always returns an empty result (floating window popup). This not only happens with the command, but also running the function  ```require('telescope.builtin').find_files()``` 

**Expected Behavior**
I expected to fill the floating window with file matches from the current working directory.

**Actual Behavior**
And empty floating window (with the corresponding divisions: Find Files, Results and Preview) 

### Details
I also tried to pass a directory to the find_files function, but I always get the same result

<details><summary>Reproduce</summary>

1. nvim -nu test.vim
2. :Telescope find_files
3. empty Telescope win

</details>

- nvim --version output: 
```
NVIM v0.5.0-dev+1227-gb518b9076
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/cc -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -O2 -g -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/home/runner/work/neovim/neovim/build/config -I/home/runner/work/neovim/neovim/src -I/home/runner/work/neovim/neovim/.deps/usr/include -I/usr/include -I/home/runner/work/neovim/neovim/build/src/nvim/auto -I/home/runner/work/neovim/neovim/build/include
Compiled by runner@fv-az207-598

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/share/nvim"

Run :checkhealth for more info
```
- Operating system: 
Artix Linux with kernel version 5.11.8
(artix linux is a fork of arch linux which replaces systemd with openrc, runit, or s6 (in my case I use openrc)
- Telescope commit: 
d367eb4

</details>

<details><summary>Configuration</summary>
<p>

```viml
set nocompatible hidden laststatus=2

if !filereadable('/tmp/plug.vim')
  silent !curl --insecure -fLo /tmp/plug.vim
    \ https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
endif

source /tmp/plug.vim
call plug#begin('/tmp/plugged')
Plug 'nvim-lua/popup.nvim'
Plug 'nvim-lua/plenary.nvim'
Plug 'nvim-telescope/telescope.nvim'
call plug#end()

autocmd VimEnter * PlugClean! | PlugUpdate --sync | close
lua << EOF

require('telescope').setup{
  defaults = {
    vimgrep_arguments = {
      'rg',
      '--color=never',
      '--no-heading',
      '--with-filename',
      '--line-number',
      '--column',
      '--smart-case'
    },
    prompt_position = "bottom",
    prompt_prefix = ">",
    selection_strategy = "reset",
    sorting_strategy = "descending",
    layout_strategy = "horizontal",
    layout_defaults = {},
    file_ignore_patterns = {},
    shorten_path = true,
    winblend = 0,
    width = 0.75,
    preview_cutoff = 120,
    results_height = 1,
    results_width = 0.8,
    border = {},
    borderchars = { '─', '│', '─', '│', '╭', '╮', '╯', '╰'},
    color_devicons = true,
    use_less = true,
  }
}
EOF
```

**Comment by tjdevries** [comment]
Do you have ripgrep installed?

**Comment by nuevasindustriasdelafantasia** [comment]
yep
``
rg --version
ripgrep 12.1.1
-SIMD -AVX (compiled)
+SIMD +AVX (runtime)
``

**Comment by tjdevries** [comment]
can you try it w/ no `vimgrep_arguments`?

**Comment by nuevasindustriasdelafantasia** [comment]
Just tried with the test.vim file without the vimgrep_arguments table and got the same result...

**Comment by nuevasindustriasdelafantasia** [comment]
hey, so, today I was working on some project in neovim and after a while I accidentally pressed the keybind to ```Telescope find_files```, which opened but this time it showed stuff on the floating  window!
At the same time my computer fans started spinning hard and neovim crashed, heheh.
So, is it possible that find_files indexes a default directory (like $HOME) recursively and doesn't show anything until it finished and for some reason on my computer it takes a while?? (my computer is a lenovo e470 with a i5 and 8gb of ram)

**Comment by duckpuppy** [comment]
I'm having a similar issue, in that all my results windows are empty. In reality - the results counter displays the correct numbers and the number seems to filter properly as I type ( updating to `14/37` for example), but the results and preview windows remain blank. This happens for all pickers.

**Comment by jswh** [comment]
> 
> 
> I'm having a similar issue, in that all my results windows are empty. In reality - the results counter displays the correct numbers and the number seems to filter properly as I type ( updating to `14/37` for example), but the results and preview windows remain blank. This happens for all pickers.

same issue here

**Comment by curio77** [comment]
I have the same issue of `find_files` not showing any results. All the while, `ps` shows an `fd` process sleeping, ` 121461 pts/3    Sl+    0:00 fd --type f`.

**Comment by nuevasindustriasdelafantasia** [comment]
Hey, soo I think I just realized what was (for me) the issue. I was under the impression that find_files wasn't working on big directories and I always tried it with my home dir since its pretty full of stuff. find_files worked fine in other folders but it was always empty when the cwd was on home.
So, I realized that I have a .gitignore on my home ignoring every file (basically just putting "*"). I do this since I set my home as a git repo to handle my dotfiles easily (this way forces you to explicitly add each file you want to the repo, and avoids a bunch of noise with files you are never going to add). 
So, if I modify that .gitignore find_files works. 
I think it took me long to realize since Telescope git_files was working on that dir (I guess its smarter that find_files and know about files that are getting tracked).
Anyways, maybe others keep having this issue, so I won't close it for know.

Also, should the find_files picker care about git? shouldn't it just grab what ever is on the directory?
